hooks/infinite/useInfiniteAccompanies.ts

```diff
@@ -0,0 +1,19 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+
+import { getCompanionsList } from '~/apis/companion';
+import { AccompanyPostInfoList } from '~/apis/scheme/accompany';
+
+const useInfiniteAccompanies = (params: string) => {
+  return useInfiniteQuery({
+    queryKey: ['accompanies', params],
+    queryFn: async ({ pageParam }): Promise<AccompanyPostInfoList> => {
```

Comment:
```suggestion
    queryFn: ({ pageParam }) => {
```
타입 지정 없이도 똑같이 추론되는것 같아요!!

Reply:
덧붙여서 아래와 같이 작성해도 좋을 것 같아요ㅎㅎ
```ts
queryFn: ({ pageParam }) => getCompanionsList(params, SIZE, pageParam)
```